src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -428,7 +428,12 @@
         "NAME_REQUIRED": "Name is required",
         "NOTIFY_TYPE": "Notify Type",
         "EVENT_TYPE": "Event Type",
-        "EVENT_TYPE_REQUIRED": "Require at least one event type"
+        "EVENT_TYPE_REQUIRED": "Require at least one event type",
+        "PAYLOAD_FORMAT": "Payload Format",
+        "CLOUD_EVENT": "Cloud Event",
```

Comment:
The Format is called **CloudEvents** not Clout Event. I strongly encourage you to change it. When I use googles for "Cloud Event" he will have a hard time finding it vs CloudEvents